Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -825,13 +820,18 @@ def addCKFTracks(
 def addTrackSelection(
     s: acts.examples.Sequencer,
     trackSelectorRanges: TrackSelectorRanges,
-    inputTrackParameters: str,
-    inputTrackParametersTips: str,
-    outputTrackParameters: str,
-    outputTrackParametersTips: str,
+    inputTrackParameters: Optional[str] = None,
```

Comment:
Hmm I'm a bit worried about this interface. It is not entirely straight forward that not providing a parameter causes the function to use a hardcoded string as alias... The minimum solution would be to document this in the docstring I would say.

However, is there a reason why not calling `resolveAlias` in the "main" script? Then the add functions would still be just dealing with reagular keys, what also would make debugging easier I guess.

This would also allow a kind of "don't use alias, so don't bother with them" style. In this case it could happen that a user is confronted with the alias system, even without using it.

Reply:
Okay actually with a second thought I totally agree with you, the add functions should try to connect to each other without need of interaction in the default case

I think it is really a matter of documentation to make the structure clear. E.g. somehow like this: "This function resolves and overwrites the aliases "trackParameters" and "trackParameterTips". To overwrite this behaviour, pass the inputTackParameters and foobar directely."

Reply:
What would be also nice is maybe a checking that gives the user useful hints: "Alias 'foo' not found. Pass 'bar' explicitly if you don't want to use the alias system" or something like that